TypeRaceAir/race.py

```diff
@@ -0,0 +1,88 @@
+# -- IMPORTS --#
+import typing
+import datetime
+
+
+class Race():
+
+    def __init__(self, sentece: str, players: typing.List[Player]):
+        '''The race sentence'''
+        self.sentece = sentece
+        '''	The players that participate in the race.'''
```

Comment:
Please use regular inline comments inside code blocks. Use docstring only at the begging of a method, function, or object. More about docstrings [here](https://www.python.org/dev/peps/pep-0257/)